src/transformers/models/longformer/modeling_tf_longformer.py

```diff
@@ -708,6 +709,16 @@ def __init__(self, config, layer_id, **kwargs):
 
         self.one_sided_attn_window_size = attention_window // 2
 
+    def build(self, input_shape=None):
+        if not self.built:
+            with tf.name_scope("query_global"):
+                self.query_global.build((self.config.hidden_size,))
+            with tf.name_scope("key_global"):
+                self.key_global.build((self.config.hidden_size,))
+            with tf.name_scope("value_global"):
+                self.value_global.build((self.config.hidden_size,))
```

Comment:
Two silly questions 
- why do these layers need to be built separately here? 
- why no `super().build(input_shape)` call in the method?